### DISCUSSION TITLE
Status of this project

### DISCUSSION THREAD

**Discussion by markedwards** [reset-button/ariadne_django#63] [General]
I see that the project has a "looking for a new maintainer" status.  Is it totally unmaintained at this point?  Is there a desire to move it to a different owner altogether?  I see that this account has no other repos.

It would be nice to see the GraphQL Playground replaced with GraphiQL 2, since the former is EOL.  Since ariadne now has the explorer class(es), maybe ariadne_django can use those instead?

**Comment by maxmorlocke** [comment]
I'm the former primary maintainer, and I'm two companies later in my work history.  If someone would like to fork and generate a new package, go for it.  I'm happy to surrender control of the pypi namespace.

**Comment by markedwards** [comment]
I'll consider it, but if someone else is keen, I'd probably defer to them.

**Comment by rafalp** [comment]
We could have this back under Mirumee's umbrella with @markedwards having the release rights for this.

@maxmorlocke can you please transfer pypi ownership of `ariadne_django` to `rafal@mirumee.com`? Could you also direct me to somebody from reset-button about repo transfer?

**Reply by maxmorlocke** [reply]
@rafalp I'm happy to turn over the repo and pypi back to Mirumee.  I can do both, just will need to do it tonight after work/kids go to sleep.

**Reply by markedwards** [reply]
I’m happy to be primary maintainer for the near term at least.  I don’t have a particularly extensive vision for development of it, but there are some things I’d like to improve.

As in, happy to have release rights on the Mirumee repo.

**Reply by maxmorlocke** [reply]
It turns out I'm no longer a maintainer here, so tagging in @bennettrogers

**Reply by bennettrogers** [reply]
Sure, I can transfer ownership of the repo to Mirumee org. @rafalp I think I just need repository creation permission?

**Reply by rafalp** [reply]
@bennettrogers you can invite me as an admin to this repository and I should be able to carry it from there.

**Reply by bennettrogers** [reply]
@rafalp Done.

**Reply by rafalp** [reply]
Thank you! Are you able to also transfer Pypi project to me?

**Comment by rafalp** [comment]
I've sent an invite to @markedwards.

@maxmorlocke @bennettrogers I still need help with transfering [the Pypi project](https://pypi.org/project/ariadne-django/). 🙏

**Reply by bennettrogers** [reply]
I'll work with @maxmorlocke to have the PyPI project transferred.

**Reply by rafalp** [reply]
Excellent, thank you! My Pypi handle is same as here: https://pypi.org/user/Rafalp/

**Reply by bennettrogers** [reply]
The transfer has been requested.

**Reply by rafalp** [reply]
It's done. Thank you for holding the candle for all those years!

**Comment by markedwards** [comment]
@rafalp - apologies that I dropped the ball here.  My use case for this integration subsided, so it fell off my radar.

**Comment by pkrakesh** [comment]
Can I use this package with the latest versions of Django? There is no updates to this project for 2 years. I don't know how to use this package with the latest default ASGI support provided by Django. Is the documentation is up to date? Anyone Kindly help.

**Reply by rafalp** [reply]
At the time of writing `ariadne-django` has no maintainers. I am expecting everything about it be outdated.
